config.ini

```diff
@@ -37,6 +37,11 @@ frontControllerClass = false
 kwc.responsive = false
 kwc.mobileBreakpoints = false
 
+
+apc.clearCacheScriptPath = false
+optcode.clearCacheScriptPath = false
+filecache.clearCacheScriptPath = false
```

Comment:
; external script for clearing optcode and apc-user cache during clear-cache
externalClearCacheScript = false